server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -143,12 +145,20 @@ public static String getBookQuery(DatabaseClient dbClient, String bookId) {
     if (resultCount == 1) {
       try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
         while (resultSet.next()) {
-          author = resultSet.getString("author");
-          isbn = resultSet.getLong("isbn");
           title = resultSet.getString("title");
+          if (resultSet.isNull("author") || resultSet.getString("author").isEmpty()) {
+            author = NO_AUTHOR;
+          } else {
+            author = "Author: " + resultSet.getString("author");
+          }
```

Comment:
if you wanted, you could change this to:
```
author = (resultSet.isNull("author") || resultSet.getString("author").isEmpty())
    ? NO_AUTHOR
    : "Author: " + resultSet.getString("author");
```

There's really no point in this other than decreased line count however.

Reply:
Thanks for the feedback, I like that change and will implement it.

Reply:
Allso, ternary operators just look clean!